### PR TITLE
Hostile ship minigame

### PR DESCRIPTION
**Pull request by jessetilro**
Implemented the Hostile Ship minigame functionality and layout. Added some icon buttons to the user interface as a replacement of some plain text buttons.

Developed (and should therefore be tested) in parallel with PR https://github.com/jessetilro/pixelperfect/pull/276

### CODE REVIEW

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/HostileShipActivity.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    private int readPositionX() {
+        EditText positionX = (EditText) findViewById(R.id.position_x);
+        return Integer.parseInt(positionX.getText().toString());
+    }
+
+    /**
+     * Read an entered integer value from the view.
+     * @return The entered integer value.
+     */
+    private int readInteger(int id) {
```
Comment (wouterz) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/HostileShipActivity.java:71]: param id javadoc

**Comment by wouterz** [comment]
Rest looks good to me

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/HostileShipActivity.java (hunk 1 above)
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/HostileShipActivity.java:71]: Fixed

File: .idea/misc.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (FlXlT) [inline .idea/misc.xml]: You forgot to discard this.

Comment (jessetilro) [inline .idea/misc.xml]: Fixed.

**Comment by dalderliesten** [comment]
Looks good to me, fansatic icons.

**Comment by FlXlT** [comment]
Nice mini-game! I agree with David, it's 'fansatic'.

David EDIT:  Floris plis, it's late.

**Comment by jessetilro** [comment]
Just a moment, will add icons for remaining minigames.

**Comment by wouterz** [comment]
fix mergeconflicts again and good to go

**Comment by dmalarev** [comment]
Looks good to me.
As soon as the conflicts are fixed, it'll have my approval.

**Pull request merged**